public/sass/_variables.dark.scss

```diff
@@ -195,7 +192,7 @@ $input-box-shadow: inset 1px 0px 0.3rem 0px rgba(150, 150, 150, 0.1);
 $input-border-focus: $input-border-color;
 $input-box-shadow-focus: rgba(102, 175, 233, 0.6);
 $input-color-placeholder: $gray-1 !default;
-$input-label-bg: $gray-blue;
+$input-label-bg: $dark-3;
```

Comment:
any reason for removing the gray blue?

Reply:
yes, I tried to merge dark-3. panel-bg and blue-gray into one color. Might have become too much compromising. Felt that panel-bg and blue-gray were to close to each other.